test/integration/linode_client/test_linode_client.py

```diff
@@ -0,0 +1,406 @@
+import re
+import time
+from test.integration.helpers import get_test_label
+
+import pytest
+
+from linode_api4 import ApiError, LinodeClient
+from linode_api4.objects import ObjectStorageKeys
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_client_and_linode(get_client):
+    client = get_client
+    available_regions = client.regions()
+    chosen_region = available_regions[0]
+    label = get_test_label()
+
+    linode_instance, password = client.linode.instance_create(
+        "g5-standard-4", chosen_region, image="linode/debian9", label=label
+    )
+
+    yield client, linode_instance
+
+    linode_instance.delete()
+
+
+def test_get_account(setup_client_and_linode):
+    client = setup_client_and_linode[0]
+    account = client.account()
+
+    assert re.search("[a-zA-Z]+", account.first_name)
+    assert re.search("[a-zA-Z]+", account.last_name)
+    assert re.search(
+        "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$", account.email
+    )
+    assert re.search(
+        "^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$", account.phone
+    )
+    assert re.search("[a-zA-Z0-9]+", account.address_1)
+    assert re.search("[a-zA-Z0-9]+", account.address_2)
+    assert re.search("[a-zA-Z]+", account.city)
+    assert re.search("[a-zA-Z]+", account.state)
+    assert re.search("[a-zA-Z]+", account.country)
+    assert re.search("[a-zA-Z0-9]+", account.zip)
+    if account.tax_id:
+        assert re.search("[0-9]+", account.tax_id)
```

Comment:
Be aware that some account may have these attribute as empty strings...

Reply:
Yeah but we are using two different ways to get account details. 
E.g.
`account = client.account()`
`account = client.load(Account, account_id)`

I updated the regex so it can check empty string as well